backend/account/views.py

```diff
@@ -97,6 +103,38 @@ def set_organization(request: Request, id: str) -> Response:
     return auth_controller.set_user_organization(request, id)
 
 
+@api_view(["GET"])
+def get_session_data(request: Request) -> Response:
+    try:
+        response = make_session_response(request)
+        return Response(
+            status=status.HTTP_201_CREATED,
+            data=response,
+        )
+
+    except Exception as error:
```

Comment:
Please try to catch specific exceptions and handle them accordingly - uncaught exceptions would anyway be logged with a traceback and returned as 500 errors by django. But we'd have more information regarding the error itself in our logs. We'll have to move out of this practice as we refactor / improve our error handling

Reply:
In this case, getting rid of the `try-catch` block would make no difference in behaviour and in-fact help improve logging with a traceback

Reply:
Removed genric excedption catch from here.